packages/@aws-cdk/app-staging-synthesizer-alpha/lib/default-staging-stack.ts

```diff
@@ -242,6 +242,7 @@ export class DefaultStagingStack extends Stack implements IStagingResources {
     super(scope, id, {
       ...props,
       synthesizer: new BootstraplessSynthesizer(),
+      description: `This stack includes resources needed to deploy the AWS CDK app ${props.appId} into this environment`,
```

Comment:
Feedback on this description is welcome.

I based this on the default `CDKToolkit` stack description:

> This stack includes resources needed to deploy AWS CDK apps into this environment

Do we want to include the term `AppStagingSynthesizer`?

Reply:
I think the current description is probably fine as is. Users will be able to easily check that resources are being created by checking the `Resources` tab anyway.

Reply:
One thing that we're not considering. The support stack (since it cannot be bootstrapped) must be uploaded to CFN directly, and CFN supports a max size of 51,200 bytes. The app support stack is dynamic because it entirely depends on how many ECR/S3 resources you want. So the description cuts into that.

Somewhere we are promising a number of ECR resources allowed per staging stack. That number *could* go down as a result of this.

That being said, we probably are ok with this.